fastai/text/models/awd_lstm.py

```diff
@@ -196,4 +198,72 @@ def awd_lstm_clas_split(model:nn.Module) -> List[nn.Module]:
                           hidden_p=0.1, input_p=0.2, embed_p=0.02, weight_p=0.15, tie_weights=True, out_bias=True)
 
 awd_lstm_clas_config = dict(emb_sz=400, n_hid=1150, n_layers=3, pad_token=1, qrnn=False, bidir=False, output_p=0.4, 
-                       hidden_p=0.2, input_p=0.6, embed_p=0.1, weight_p=0.5)
\ No newline at end of file
+                       hidden_p=0.2, input_p=0.6, embed_p=0.1, weight_p=0.5)
+
+# Interpretability
+
+import matplotlib.cm as cm
+
+def value2rgba(x:float, cmap:Callable=cm.RdYlGn, alpha_mult:float=1.0)->Tuple:
+    "Convert a value `x` from 0 to 1 (inclusive) to an RGBA tuple according to `cmap` times trasparency `alpha_mult`."
+    c = cmap(x)
+    rgb = (np.array(c[:-1]) * 255).astype(int)
+    a = c[-1] * alpha_mult
+    return tuple(rgb.tolist() + [a])
+
+def piece_attn_html(pieces:List[str], attns:List[float], sep:str=' ', **kwargs)->str:
+    from html import escape
+    html = []
+    html.append('<span style="font-family: monospace;">')
```

Comment:
Initial html to this to save a line.
Even better, initialize spans at the same time:
```
html,spans = ['<span...>'],[]
```